submissions/SofiiaTrokhymchuk/tiny-js-world/index.js

```diff
@@ -50,12 +50,8 @@ catWoman.legs = 2;
 catWoman.hands = 2;
 
 function inhabitantToString(inhabitant){
-   const friends = [];
-   if(inhabitant.friends === undefined){
-      friends.push(0);
-   }else{
-      friends.push(...inhabitant.friends);
-   }
+   let friends = [];
+   inhabitant.friends === undefined ? friends[0] = 0 : friends = inhabitant.friends;
```

Comment:
1. Proper use of [Ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator)
2. `[0]` is a legitimate assignable value.
3. These 2 statements can be a single statements.

Reply:
You mean, that I need to combine a declaration and initialisation of `friends` and ternary operator into a single statement?

Reply:
Yes